doc/conf.py

```diff
@@ -0,0 +1,115 @@
+
+
+import sys
+import os
+import sphinx_rtd_theme
+import recommonmark
+
+from recommonmark.transform import AutoStructify
+from os.path import abspath, join, dirname
+
+sys.path.insert(0, abspath(join(dirname(__file__))))
+
+# -- RTD configuration ------------------------------------------------
+
+on_rtd = os.environ.get("READTHEDOCS", None) == "True"
+
+# This is used for linking and such so we link to the thing we're building
+rtd_version = os.environ.get("READTHEDOCS_VERSION", "latest")
+if rtd_version not in ["stable", "latest"]:
+    rtd_version = "stable"
+
+# -- Project information -----------------------------------------------------
+
+project = 'cardano-graphql Documentation'
+copyright = '2020, IOHK'
+author = 'IOHK'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0.0'
```

Comment:
Is this version meant to be independent of the software version?

Reply:
Yes, the software version is related to specific commits, not something specified in the conf file

Reply:
@rhyslbw I think this is resolved right?